syft.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: syft
+  version: 0.70.0
+  epoch: 0
+  description: CLI tool and library for generating a Software Bill of Materials from container images and filesystems
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - apk-tools
+
+environment:
+  contents:
+    packages:
+      - git
```

Comment:
Do we need `git`?

Reply:
Yea, go install fetches things with git.

Reply:
Well, I guess it doesn't always, but it can. I'll try without and see if it works here!

Reply:
nvm - it's installed anywhere here: https://github.com/chainguard-dev/melange/blob/main/pkg/build/pipelines/go/install.yaml#L8